### PR TITLE
New organization

### PR DESCRIPTION
**Pull request by sabrinatoro**
@nicolevasilevsky @matentzn 
Please check this new proposed organization.
Note that it looks like we have some pages which might contain similar information, but this is future work.

### CODE REVIEW

**Review by nicolevasilevsky — APPROVED** [review]
this looks good to me, thank you!
Feel free to go ahead and merge

File: mkdocs.yml [hunk 1]
```diff
@@ -8,40 +8,45 @@ markdown_extensions:
...
-      - Bulk Editing Using Regular Expressions: editors-guide/regex.md
+      - Qualtiy control tests: editors-guide/quality-control-tests.md
+      - Generate reports: editors-guide/generate-report.md
+      - Check for and Merge Duplicate annotations in Mondo: editors-guide/NORM.md
       - Risky obsoletion check: editors-guide/risky_obsoletion_check.md
-      - Merging a ROBOT template into Mondo: editors-guide/robot-template.md
-      - Set up new computer: editors-guide/set-up-new-computer.md
-      - SOP Splitting Terms: editors-guide/sop_splitting-terms.md
-      - Splitting classes: editors-guide/splitting-classes.md
+      - Difs: editors-guide/diffs.md
```
Comment (matentzn) [inline mkdocs.yml:43]: Diffs

File: mkdocs.yml [hunk 2]
```diff
@@ -8,40 +8,45 @@ markdown_extensions:
...
+    - Troubleshooting:
       - Troubleshooting: editors-guide/troubleshooting.md
-      - Update external sources: editors-guide/update-external-sources.md
-    - Design patterns:
-      - Design patterns: editors-guide/e-design-patterns.md
-      - Patterns: editors-guide/patterns/
+      - GitHub conflicts: editors-guide/dd-github-conflicts.md
+      - Repair obsoleted classes: developer-guide/repair-obsoleted-classes.md
+      - Debugging Docs: developer-guide/debugging-docs.md      
+    - Miscelaneous:
```
Comment (matentzn) [inline mkdocs.yml:49]: Miscellaneous

File: mkdocs.yml [hunk 3]
```diff
@@ -8,40 +8,45 @@ markdown_extensions:
...
-      - Qualtiy control tests: editors-guide/quality-control-tests.md
+      - Release workflow: developer-guide/release.md
+      - Imports: editors-guide/imports.md
+      - Import terms for logical axioms: editors-guide/import-terms-for-logical-axioms.md
+      - Update external sources: editors-guide/update-external-sources.md
+      - OMIM slurp: editors-guide/omim-slurp.md
+    - Quality Control and Checks (QC):
       - Quality control: editors-guide/quality-control.md
-      - Bulk Editing Using Regular Expressions: editors-guide/regex.md
+      - Qualtiy control tests: editors-guide/quality-control-tests.md
```
Comment (matentzn) [inline mkdocs.yml:39]: Quality

**Review by matentzn — CHANGES_REQUESTED** [review]
I am missing some more clear user facing section including the obsoletion guidelines and the new mapping docs we just wrote :)

**Comment by sabrinatoro** [comment]
@matentzn 
I resolved the typos issues (though it looks like it was sent to another PR... )
Note that I simply moved things around, so there might be more typos... 

The new document was not yet merged when I did this. I don't know how that will affect @nicolevasilevsky's PR. 

In terms of "user facing section" : I created the section "Mondo Editing (rules and practices)"
Do you mean that we need documentation for users to refer to in order to learn our workflow, how we handle obsoletion,... ? 
Maybe we need to create new documentation especially created for users? (I honestly don't know if any of the one we currently have is clearly user centric - except for the new one which is not here)

**Comment by matentzn** [comment]
The ones that explain what the annotation properties / codes mean we use in Mondo, the one with the obsoletion contract, the mapping one anything that is for people that _use_ mondo for something, like clingen.

**Pull request merged**